Samples/Server/NativeServersUnitTests/NativeServersTests.cpp

```diff
@@ -25,6 +33,145 @@ namespace NativeServersUnitTests
+
+			// Connect to the root\cimv2 namespace with
+			// the current user and obtain pointer pSvc
+			// to make IWbemServices calls.
+			hres = pLoc->ConnectServer(
+				_bstr_t(L"ROOT\\CIMV2"), // Object path of WMI namespace
+				NULL,                    // User name. NULL = current user
+				NULL,                    // User password. NULL = current
+				0,                       // Locale. NULL indicates current
+				NULL,                    // Security flags.
+				0,                       // Authority (for example, Kerberos)
+				0,                       // Context object 
+				&pSvc                    // pointer to IWbemServices proxy
+			);
+
+			Assert::IsFalse(FAILED(hres));
+
+			// Step 5: --------------------------------------------------
+			// Set security levels on the proxy -------------------------
+
+			hres = CoSetProxyBlanket(
+				pSvc,                        // Indicates the proxy to set
+				RPC_C_AUTHN_WINNT,           // RPC_C_AUTHN_xxx
+				RPC_C_AUTHZ_NONE,            // RPC_C_AUTHZ_xxx
+				NULL,                        // Server principal name 
+				RPC_C_AUTHN_LEVEL_CALL,      // RPC_C_AUTHN_LEVEL_xxx 
+				RPC_C_IMP_LEVEL_IMPERSONATE, // RPC_C_IMP_LEVEL_xxx
+				NULL,                        // client identity
+				EOAC_NONE                    // proxy capabilities 
+			);
+
+			Assert::IsFalse(FAILED(hres));
+			
+			// Step 6: --------------------------------------------------
+			// Use the IWbemServices pointer to make requests of WMI ----
+
+			// For example, get the name of the operating system
+			IEnumWbemClassObject* pEnumerator = NULL;
+			hres = pSvc->ExecQuery(
+				bstr_t("WQL"),
+				bstr_t("SELECT * FROM Win32_VideoController"),
+				WBEM_FLAG_FORWARD_ONLY | WBEM_FLAG_RETURN_IMMEDIATELY,
+				NULL,
+				&pEnumerator);
+			
+			Assert::IsFalse(FAILED(hres));
+
+			// Step 7: -------------------------------------------------
+			// Get the data from the query in step 6 -------------------
+
+			IWbemClassObject *pclsObj = NULL;
+			ULONG uReturn = 0;
+			
+			VARIANT driverNumber; //Store the driver version installed
+			bool NvidiaPresent = false; //Flag for Nvidia card being present
+
+			while (pEnumerator)
+			{
+				HRESULT hr = pEnumerator->Next(WBEM_INFINITE, 1,
+					&pclsObj, &uReturn);
+
+				if (0 == uReturn)
+				{
+					break;
+				}
+
+				VARIANT vtProp;
+				//Finds the manufacturer of the card
+				hr = pclsObj->Get(L"AdapterCompatibility", 0, &vtProp, 0, 0);
+				
+				//Find the nvidia card
+				if (!wcscmp(vtProp.bstrVal, L"NVIDIA")) {
+					//Set the Nvidia card flag to true
+					NvidiaPresent = true;
+
+					hr = pclsObj->Get(L"DriverVersion", 0, &driverNumber, 0, 0);
+					wchar_t *currentDriver = driverNumber.bstrVal;
+					size_t len = wcslen(currentDriver);
+					
+					//Major version number of the card is found at the -7th index
+					std::wstring majorVersion(currentDriver, len - 6, 1);
+					
+					//All drivers from 3.0 onwards support nvencode
```

Comment:
Can you share me the documents about "All drivers from 3.0 onwards support nvencode"?

Reply:
This is just extrapolation based on this file: https://github.com/CatalystCode/3dtoolkit/blob/master/Libraries/WebRTC/3dtoolkit_upgrades.patch#L1175

Here the check is to see if the (major version * 16) < 48, i.e. that the major version is 3 or higher.